taskfile/ast/task.go

```diff
@@ -219,7 +221,7 @@ func (t *Task) DeepCopy() *Task {
 		Platforms:            deepcopy.Slice(t.Platforms),
 		Location:             t.Location.DeepCopy(),
 		Requires:             t.Requires.DeepCopy(),
-		Namespace:            t.Namespace,
+		Namespace:            append([]string{}, t.Namespace...),
```

Comment:
This change isn't necessary.

```suggestion
		Namespace:            t.Namespace,
```

Reply:
But this is not an array, which would mean that a deep copy would need to deep copy the array too right?

Reply:
Ah yes sorry, you are right. We have `deepcopy.Slice()` for this.

```suggestion
		Namespace:            deepcopy.Slice(t.Namespace),
```